main.py

```diff
@@ -1,12 +1,189 @@
-    pass
-
+    print("\n❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁\n")
+    print("✯✯✯✯✯✯ 𝕎𝔼𝕃ℂ𝕆𝕄𝔼 𝕋𝕆 ℝ𝔼𝕋ℝ𝕆 𝕍𝕀𝔻𝔼𝕆 𝕊𝕋𝕆ℝ𝔼 ✯✯✯✯✯✯")
+    print("\n❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁❁\n")
 
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
+
+def print_stars():
+    print("\n******************************************\n")
+
+def list_options():
+
+    options = {
+        "1": "add a video", 
+        "2": "edit a video",
+        "3": "delete a video", 
+        "4": "get information about all videos", 
+        "5": "get information about one video", 
+        "6": "add a customer",
+        "7": "edit a customer",
+        "8": "delete a customer",
+        "9": "get information about all customers",
+        "10": "get information about one customers",
+        "11": "check out a video to a customer",
+        "12": "check in a video from a customer",
+        "13": "List all options",
+        "14" : "Quit"
+        }
+
+    print_stars()
+    print("✔✔✔These are the actions you can perform at RETRO VIDEO STORE✔✔✔")
+    print_stars()
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_stars()
+
+    return options
+
+def make_choice(options, video_store):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do? Please make a selection.")
+        choice = input("Make your selection using the option number: ")
+    return choice
+
+def run_cli(play=True):
+
+    #initialize video store
+    video_store = VideoStore(url=BACKUP_URL)
+
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, video_store)
+
+# ========VIDEO OPTIONS======== #
+
+        # Option 1: add a video
+        if choice=='1':
+            print("Let's create a new video.")
+            new_title = input("Title: ")
+            new_release_date = input("Release Date: ")
+            new_total_inventory = input("Total Inventory: ")
+            response = video_store.add_video(title=new_title, release_date=new_release_date, total_inventory=new_total_inventory)
+
+            print(f"New video #{response['id']} has been added.")
+
+        # Option 2: edit a video 
+        elif choice=='2':
+            select_video = input("Please select the video to edit by entering its id: ")
+            if select_video.isnumeric():
+                title=input("What is the new title of your video? ")
+                release_date=input("What is the new release date of your video? ")
+                total_inventory=input("What is the new total inventory of your video? ")
+                response = video_store.update_video(select_video,title=title, release_date=release_date, total_inventory=total_inventory)      
+
+                print(f"Video #{response['id']} has been updated.") 
```

Comment:
This line will error out when the response has an error message because the response will presumable not have an "id" field in the body in that case. (See my comment on the update_video method for more details